projects/batfish/src/main/java/org/batfish/representation/cumulus/BgpPassiveNeighbor.java

```diff
@@ -0,0 +1,18 @@
+package org.batfish.representation.cumulus;
+
+import javax.annotation.Nonnull;
+import org.batfish.datamodel.Prefix;
+
+/** Passive BGP neighbor identified by a listen range prefix. */
+public class BgpPassiveNeighbor extends BgpNeighbor {
```

Comment:
I'd call it BgpDynamicNeighbor instead to avoid confusion with the `passive` command in the future.

Reply:
What exactly does the `passive` command do? Is it independent of the neighbor being dynamic? 

Related: Can an unnumbered peer be passive?

Reply:
Yes, it's absolutely independent command. Can be unnumbered as well.